unitary/examples/quantum_rpg/classes.py

```diff
@@ -0,0 +1,36 @@
+from typing import Any, Callable, Dict
+
+from unitary import alpha
+from unitary.examples.quantum_rpg import qaracter
+
+
+class Engineer(qaracter.Qaracter):
+    """Example player class that can do an X gate.
+
+    Engineeers can do an X gate at level 1 and a H gate at level 3.
```

Comment:
Just curious, how is an H gate more powerful than X in a battle?

Reply:
I am not sure it is more powerful.  This is just an example, but I was thinking that the first few battles would essentially be 'classical', and the Hadamard allows you to handle enemies that are in the Clifford space.